project-fob/Controllers/HomeController.cs

```diff
@@ -220,5 +158,39 @@ public static string GenerateIdByLength(int length)
             return new string(Enumerable.Repeat(chars, length).Select(s => s[random.Next(s.Length)]).ToArray());
         }
 
+        public void CheckCookies()
+        {
+            string id = GetCookieID();
+            SetCookie(id);
+        }
+
+        public string GetID()
+        {
+            string id = GetCookieID();
+            SetCookie(id);
+            return id;
+        }
+
+        private string GetCookieID()
+        {
+            return Request.Cookies["ID"];
+        }
```

Comment:
When should GetID() be used and when should GetCookieID be used? It's not clear to me from the method names.